ATTIC/deployment/local/Tiltfile

```diff
@@ -1,9 +1,9 @@
-# -*- mode: python; -*-
 k8s_yaml(kustomize('.'))
-docker_build('raiinbow/webapp', '../../../apps/webapp',
+docker_build('gcr.io/k8s-staging-apisnoop/webapp', '../../../apps/webapp',
```

Comment:
Just wanting to note that this is what you want merged?

Reply:
yah!  This is the standard place we are keeping the apisnoop docker images now, yah?